cvcl/forms.py

```diff
@@ -47,15 +48,19 @@ def process_data(self, course_id):
         f = io.TextIOWrapper(self.cleaned_data['data_file'].file)
         reader = csv.DictReader(f)
 
-        for user in reader:
-            email = user['Username'] + "@drexel.edu"
-            username = user['Username']
+        for user1 in reader:
 
+            user1 = {re.sub("['\"]", "", str(k.encode('ascii', 'ignore')))[1:]: v for k, v in user1.items()}
+
+            username = user1['Username']
+            email = user1['Username'] + getattr(settings, "EMAIL_DOMAIN", None)
```

Comment:
What if you get "None"?
`TypeError: Can't convert 'NoneType' object to str implicitly`